pkg/stacks/unpack.go

```diff
@@ -44,6 +45,15 @@ const (
 	groupFileName           = "group.yaml"
 	appFileName             = "app.yaml"
 	stackFileName           = "stack.yaml"
+	behaviorFileName        = "behavior.yaml"
```

Comment:
i'm getting confused between app.yaml, stack.yaml, and behavior.yaml also now...it would be really nice to have the [stack package format "spec"](https://github.com/crossplaneio/crossplane/blob/master/design/design-doc-stacks.md#stack-package-format) updated with explanations of all these different files.

Similar to our stack CRD situation right now, some of these files may be transient/temporary, which only causes more confusion as well.  The process around "experimental" features has continued to make me feel somewhat uncomfortable, with really low code coverage, temporary types and code, etc.  I'm definitely looking forward to discussing and solidifying as a community what our process is here that we are all comfortable with in #1174. 

/cc @suskin @muvaf @negz